src/common/ga.js

```diff
@@ -0,0 +1,25 @@
+import { waitForGlobal } from './utils';
+
+let client;
+const getClient = () => {
```

Comment:
async/await would make this less... nesty

Reply:
Agree